### PR TITLE
only capitalized fixmes

### PR DESCRIPTION
**Pull request by ABaldwinHunter**
cc @codeclimate/review 

Note: Some additional tests seem good to add, but might require some refactoring of the code. I started working on that in a separate branch, and think it's better as a separate PR.

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -11,4 +11,3 @@ exclude_paths:
 - "Dockerfile"
 - "bin/fixme"
 - "tests/**"
-- ".codeclimate.yml"
```
Comment (GordonDiggs) [inline .codeclimate.yml]: Why this change?

**Comment by GordonDiggs** [comment]
One minor question. LGTM

File: .codeclimate.yml (hunk 1 above)
Comment (ABaldwinHunter) [inline .codeclimate.yml]: Because https://github.com/codeclimate/codeclimate-fixme/commit/ee8d1cf27b2c9612cd8bc9aa17175dd7eb438167

Essentially we don't need it now. But happy to keep if that seems preferable?

Comment (GordonDiggs) [inline .codeclimate.yml]: Makes sense. I'm cool with removing it
